ggml-alloc.c

```diff
@@ -268,7 +268,7 @@ struct ggml_allocr * ggml_allocr_new(void * data, size_t size, size_t alignment)
         /*.parse_seq     = */ {0},
         /*.parse_seq_len = */ 0,
 #ifdef GGML_ALLOCATOR_DEBUG
-        /*.allocated_tensors = */ = {0},
+        /*.allocated_tensors = */ {0},
```

Comment:
There is another case too. It would be good to sync all the changes to ggml-alloc from https://github.com/ggerganov/ggml/pull/476